xml/System.Text.Json/JsonProperty.xml

```diff
@@ -211,9 +211,13 @@ This method is functionally equal to doing an ordinal comparison of `text` and <
         <Parameter Name="writer" Type="System.Text.Json.Utf8JsonWriter" />
       </Parameters>
       <Docs>
-        <param name="writer">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="writer">The writer into which the property is to be written.</param>
```

Comment:
@rpetrusha modified the `writer` param in my other PRs to look like below, so I'll apply it:

```suggestion
        <param name="writer">The writer to which to write the property.</param>
```